lexoid/core/parse_type/static_parser.py

```diff
@@ -295,3 +298,42 @@ def parse_with_pdfplumber(path: str, raw: bool, **kwargs) -> List[Dict] | str:
         }
         for page_num, page_text in enumerate(page_texts, start=1)
     ]
+
+
+def process_pdf_with_pdfplumber(path: str) -> List[str]:
```

Comment:
Possible to add a test for these changes as well?

Reply:
Done